src/clarity-angular/emphasis/alert/alert.module.ts

```diff
@@ -21,6 +21,9 @@ import { ALERT_DIRECTIVES } from "./index";
     ],
     exports: [
         ALERT_DIRECTIVES
+    ],
+    schemas: [
```

Comment:
@mathisscott: Importing `ClrIconModule` should be enough. Unless you're expecting consumers to write `<clr-icon>` in their own template, in which case you might want to also export `ClrIconModule`. But this schema is actually a blocker, it hides too many errors to ship with it.